src/controller/python/chip/ChipBleUtility.py

```diff
@@ -0,0 +1,300 @@
+#
+#    Copyright (c) 2020 Project CHIP Authors
+#    Copyright (c) 2019-2020 Google LLC.
+#    Copyright (c) 2015-2018 Nest Labs, Inc.
+#    All rights reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+#
+#    @file
+#      This file is utility for Chip BLE
+#
+
+from __future__ import absolute_import
+from __future__ import print_function
+from ctypes import *
+from .ChipUtility import ChipUtility
+import six
+
+
+# Duplicates of BLE definitions in ChipDeviceController-ScriptBinding.cpp
+BleEventType_Rx                             = 1
+BleEventType_Tx                             = 2
+BleEventType_Subscribe                      = 3
+BleEventType_Disconnect                     = 4
+
+BleSubscribeOperation_Subscribe             = 1
+BleSubscribeOperation_Unsubscribe           = 2
+
+# From BleError.h:
+BLE_ERROR_REMOTE_DEVICE_DISCONNECTED        = 12
```

Comment:
some constants here are lowercase and some camelcase. Should they be uniform? or are we matching casing for enum vs #defines?

Reply:
Just make them uniform, we should follow python style. thanks for catching them.